src/charm.py

```diff
@@ -309,6 +341,15 @@ def _unit_ips(self) -> List[str]:
         addresses.append(self_address)
         return addresses
```

Comment:
The return type is `list` but the property name and type hint indicate `bool`. What is the intention?